Tools/requirements-hypothesis.txt

```diff
@@ -1,4 +1,5 @@
 # Requirements file for hypothesis that
 # we use to run our property-based tests in CI.
 
+attrs<=23.2.0
```

Comment:
unrelated change

Reply:
was a quick test for https://github.com/python/cpython/pull/122687, sorry; reverted